### PR TITLE
Update README.md to describe how we collect telemetry

### PR DESCRIPTION
**Pull request by sapohl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,25 @@
 # outofdate-notifications-system-addon
-This system addon notifies users that they are out of date via a doorhanger in the browser
+This system addon notifies users that they are out of date via a doorhanger
+notification in the browser.
 
 ## Telemetry
 This addon sends telemetry pings when it starts, when the doorhanger is shown,
```
Comment (georgf) [inline README.md]: Nit: Could format this as a list to make the individual events stand out clearer.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,25 @@
...
-      clicked: <bool>
+      event: "started"|"shown"|"clicked"
     }
 
-The two flags indicate whether the doorhanger notification has been shown, and
-whether the button linking to the download page has been clicked.
-The optional environment data from the common ping format is not included,
-but the optional client ID is included, so that these pings can be correlated
-with other telemetry.
+The event indicates when the addon has started up, the doorhanger notification
```
Comment (georgf) [inline README.md]: Nit: This could be clearer as a list ("'started' - indicates that ...", etc.).

**Comment by georgf** [comment]
r=me with this addressed.

**Comment by sapohl** [comment]
@georgf: sending back for a final look. Thanks!

**Comment by georgf** [comment]
Looks good, thanks.

**Pull request merged**